BREAKING.md

```diff
@@ -952,18 +951,14 @@ As part of the Fluid Data Library (FDL) and Fluid Component Library (FCL) split
 -   `IComponentSerializer `will become `IFluidSerializer`
 -   `IComponentTokenProvider` will become `IFluidTokenProvider`
 
-`IComponent` will also become `IFluidObject`, and the mime type for for requests will change from `fluid/component` to `fluid/object`
```

Comment:
Let's not change BREAKING.md, these notes are still accurate for the versions they refer to.

Reply:
Done!